roles/gaia/tasks/main.yml

```diff
@@ -66,7 +66,18 @@
   # TODO: should we overwrite?
   shell: |
     PATH=$PATH:/usr/local/go/bin:{{ ansible_env.HOME }}/go/bin
-    gaiad init {{inventory_hostname}} --home {{gaiad_home}}
+    gaiad init {{inventory_hostname}} --home {{gaiad_home}} --chain-id "theta-devnet"
```

Comment:
Could we have `{{chain_id}}` here instead and add the default name to the vars?

Reply:
yes, good to do